docs/integrations/tensorflow.rst

```diff
@@ -0,0 +1,91 @@
+Tensorflow Integration
+-----------------------
+
+Lance can be used as a regular `tf.data.Dataset <https://www.tensorflow.org/api_docs/python/tf/data/Dataset>`_
+in `Tensorflow <https://www.tensorflow.org/>`_.
+
+.. warning::
+
+    This feature is experimental and the APIs may change in the future.
+
+Reading from Lance
+~~~~~~~~~~~~~~~~~~
+
+Using :meth:`lance.tf.data.from_lance`, you can create an `tf.data.Dataset` easily.
+
+.. code-block:: python
+
+    import tensorflow as tf
+    import lance
+
+    # Create tf dataset
+    ds = lance.tf.data.from_lance("s3://my-bucket/my-dataset")
+
+    # Chain tf dataset with other tf primitives
+
+    for batch in ds.shuffling(32).map(lambda x: tf.io.decode_png(x["image"])):
+        print(batch)
+
+Backed by the Lance `columnar format <../format.rst>`_, using :meth:`lance.tf.data.from_lance` supports
+efficient column selection, filtering, and more.
+
+.. code-block:: python
+
+    ds = lance.tf.data.from_lance(
+        "s3://my-bucket/my-dataset",
+        columns=["image", "label"],
+        filter="split = 'train' AND collected_time > timestamp '2020-01-01'",
+        batch_size=256)
+
+By default, Lance will infer the Tensor spec from the projected columns. You can also specify :class:`tf.TensorSpec` manually.
+
+.. code-block:: python
+
+    batch_size = 256
+    ds = lance.tf.data.from_lance(
+        "s3://my-bucket/my-dataset",
+        columns=["image", "label"],
+        batch_size=batch_size,
+        output_signature={
+            "text": tf.TensorSpec(shape=(), dtype=tf.string),
```

Comment:
The column uses "image", but the specification here is "text".

Reply:
done